test/e2e/stack.go

```diff
@@ -89,12 +99,17 @@ func (e *e2eCfnStack) deployStack(ctx context.Context, logger logr.Logger) error
 			ParameterValue: aws.String(string(e.iamRolesAnywhereCACert)),
 		},
 	}
-
+	var buf bytes.Buffer
+	cfnTemplate := template.Must(template.New("hybridCfnTemplate").Parse(cfnTemplateBody))
```

Comment:
is there a reason to use `must` instead of checking for an error and returning it?

Reply:
Thanks, I think creating template without must and handle the error manually makes more sense. Removed Must